pysat/tests/test_instrument.py

```diff
@@ -174,6 +174,30 @@ def test_filtered_date_attribute(self):
         self.testInst.date = now
         assert today == self.testInst.date
 
+
```

Comment:
Relevant changes below.

Reply:
This comment no longer needed now that the ICON updates have been properly merged. All code differences are authentic now.